models/issuetrackers/pivotaltracker.coffee

```diff
@@ -316,6 +327,15 @@ tryFailTesting = (event) ->
   original_labels = event.original_labels
   new_labels = event.new_labels
 
+  isPassedReviewLabel = (label) -> label in [reviewedLabel, noReviewLabel]
+  isFailedTestingLabel = (label) -> label is failedLabel
```

Comment:
Do you need to duplicate this code? Couldn't you just move those function to the module scope?